### PR TITLE
GUVNOR-3200: Cover edit column permissions for users with multiple roles

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

**Comment by jomarko** [comment]
@karreiro I think something for your eyes.

File: uberfire-security/uberfire-security-api/src/test/java/org/uberfire/security/impl/authz/DefaultPermissionManagerTest.java [hunk 1]
```diff
@@ -54,71 +55,191 @@ public void setUp() {
...
-        }};
-        final HashSet<Group> groups = new HashSet<Group>() {{
-            // Users have a group with their names by default
-            add(new GroupImpl(name));
-        }};
+                              final Group group,
+                              final Role... roles) {
 
-        return new UserImpl(name, roles, groups);
+        return new UserImpl(name, Arrays.asList(roles), Arrays.asList(group));
```
Comment (karreiro) [inline uberfire-security/uberfire-security-api/src/test/java/org/uberfire/security/impl/authz/DefaultPermissionManagerTest.java]: I think that we could use `Collections.singletonList(group)` here :-)

Comment (jomarko) [reply uberfire-security/uberfire-security-api/src/test/java/org/uberfire/security/impl/authz/DefaultPermissionManagerTest.java]: Sure, no problem, I will apply and let you know once incorporated, thank you for looking.

**Comment by jomarko** [comment]
@karreiro I have added separate commit.

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by karreiro** [comment]
Jenkins please execute full downstream build.

**Comment by ederign** [comment]
@karreiro feel free to merge this when it's ok for you.

**Pull request merged**